src/ngx_http_lua_ssl.c

```diff
@@ -34,4 +34,35 @@ ngx_http_lua_ssl_init(ngx_log_t *log)
 }
 
 
+int
+ngx_http_lua_ssl_password_callback(char *buf, int size, int rwflag,
```

Comment:
This should probably be moved to `ngx_http_lua_ssl_certby.c` and declared static since it's only used there.